.circleci/config.yml

```diff
@@ -50,7 +50,7 @@ commands:
       - run:
           name: "Install Grafana build pipeline tool"
           command: |
-            VERSION=0.2.8
+            VERSION=0.2.9-phantomjs-2
```

Comment:
Can you release v0.2.9 of the build pipeline tool now, and update this?

Reply:
Will do that before merging removal of phantomjs feature branch to master. Or am I missing something?

Reply:
@marefr I don't think you are missing anything :) I didn't realize this PR is targeting another branch than master.